docs/articles/documentation/reference/test-api/testcontroller/getcurrentwindow.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: docs
+title: t.getCurrentWindow Method
+permalink: /documentation/reference/test-api/testcontroller/getcurrentwindow.html
+---
+
+# t.getCurrentWindow method
+
+Retrieves a `window` object that corresponds to the currently open window.
+
+```js
+t.getCurrentWindow();
```

Comment:
Why not let `openWindow` return a window object? As far as I can tell this eliminates necessity for this method.

Reply:
Windows can also be opened client-side (without evoking the t.openWindow method)

Reply:
Makes, but here in example `openWindow` is explicitly followed by `getCurrentWindow` which leaves an impression that it's the intended way to obtain a window handle.

Reply:
@inikulin `openWindow` returns window object as well

Reply:
We can use `getCurrentWindow` when we want to get window handler of the first window, which was not opened via the `openWindow` method

Reply:
yeah, that's my point. Otherwise, looking at the docs, users may assume that they need to always call `getCurrentWindow` after `openWindow` to acquire a window handle, which is far from being ergonomic.